core/modules/mysql/testLocalInfile.cc

```diff
@@ -48,33 +48,37 @@ class Api {
 public:
     Api() {
         mysql_init(&cursor);
+		mysql_options( &cursor, MYSQL_OPT_LOCAL_INFILE, 0 );
     }
     ~Api() {
         mysql_close(&cursor);
     }
 
     void connect() {
-        MYSQL* cur = mysql_real_connect(&cursor,
-                                        "localhost", // host
-                                        "danielw", // user
-                                        "", // pw
-                                        "", // db
-                                        0, // port
-                                        "/u1/local/mysql.sock", // socket
-                                        0); // client flag
-        if(!cur) {
-            std::cout << "Failed to connect to MySQL: Error: "
-                      << mysql_error(cur) << std::endl;
-            assert(cur);
+        mysql_real_connect(&cursor,
+				"localhost", // host
+				"qsmaster", // user
+				"", // pw
+				"", // db
+				0, // port
+				"/home/qserv/qserv-run/git/var/lib/mysql/mysql.sock", // socket
+				0); // client flag
+        if(!&cursor) {
```

Comment:
This test will never succeed, `&cursor` can never be NULL, it's an address of member variable. You need to return back `MYSQL* cur = mysql_real_connect(...)`, otherwise there is no way to test for failure.